server/src/Q3Server/Startup.cs

```diff
@@ -25,13 +29,24 @@ public void Configuration(IAppBuilder app)
 
             var manager = new QManager(new QEventsListener(GlobalHost.ConnectionManager));
 
+            var userGetterCombined = new ObjectGetterTernary<User>(new UserGetterDomain(),

```

Comment:
I don't think that this ObjectGetterTernary class is justified in its generality I'm afraid - we're only using it once, and the condition is "does the first getter return null?" :) Please could you change this to e.g. just be a UserGetter class, still inheriting from IObjectGetter<User>, but explicitly containing the logic of "try Domain then Serialized"? Thanks